src/http.facility.js

```diff
@@ -43,9 +44,18 @@ class HttpFacility extends Base {
         opts = {}
       }
 
-      const url = path.includes('://') ? path : `${this.baseUrl}/${path.replace(/^\//, '')}`
+      let url = path.includes('://') ? path : `${this.baseUrl}/${path.replace(/^\//, '')}`
+
+      const urlHasQuestion = url.includes('?')
+      if (this.qs) url += (urlHasQuestion ? '' : '?') + this.qs
```

Comment:
if the url has `?` and we have non-empty default `qs`, we should add it with `&`, I think